resources/views/moderator/schedule/index.blade.php

```diff
@@ -0,0 +1,116 @@
+@php use Carbon\Carbon; @endphp
+<x-app-layout>
+    <x-slot name="header">
+        <h1 class="font-semibold text-3xl text-gray-800 dark:text-gray-200 leading-tight">
+            Schedule management
+        </h1>
+        <h2 class="font-semibold text-2xl text-gray-800 dark:text-gray-200 leading-tight">
+            General overview
+        </h2>
+    </x-slot>
+
+    <div class="pb-12">
+        <div class="max-w-7xl mx-auto py-5 mt-12 sm:px-6 lg:px-8 dark:bg-gray-800 bg-gray-200 rounded">
+            <div class="grid grid-cols-8 gap-4 pb-12">
+                <div style="grid-column: span 3;">
+                    <h2 class="text-2xl text-gray-800 dark:text-gray-200 leading-tight">
+                        Current version of schedule
+                    </h2>
+                </div>
+                <div>
+                    <a href="{{route('moderator.requests', 'presentations')}}"
+                       class="bg-purple-800 text-xs text-white py-2 px-4 rounded block text-center transition-all duration-300 transform hover:scale-105">
+                        {{$numberOfPresentationRequest}}<br>Presentations<br>waiting to be approved
```

Comment:
Using `<br>` is the old way of doing it. Preferred way is using `<div>` or `<p>` I think tailwind has some classes for lineheight as well. You can also leave this be for a frontend refactor round.